pkgs/top-level/all-packages.nix

```diff
@@ -3406,7 +3406,7 @@ with pkgs;
 
   lepton-eda = callPackage ../applications/science/electronics/lepton-eda { };
 
-  lexicon = callPackage ../tools/admin/lexicon { };
```

Comment:
We try to avoid mixing of package scopes, it should be using the `pkgs.callPackage`

Reply:
But building it this way wouldn't recognize buildPythonApplication:

```
·> nix-build -A lexicon
error: anonymous function at /home/syane/Projektujo/Aliaj/nixpkgs/pkgs/tools/admin/lexicon/default.nix:1:1 called without required argument 'buildPythonApplication'

       at /home/syane/Projektujo/Aliaj/nixpkgs/lib/customisation.nix:69:16:

           68|     let
           69|       result = f origArgs;
             |                ^
           70|
```

Reply:
With https://github.com/NixOS/nixpkgs/pull/163010/files#r824255580 this change is not needed.

Reply:
Done